### PR TITLE
Button

### PR DESCRIPTION
**Pull request by ooreally**
feat: added button to component library
Here is the deploy link: https://deploy-preview-6--decora.netlify.app/components/button/button.html
Here is the screenshot.
 
image

### CODE REVIEW

**Review by arulk8 — COMMENTED** [review]
Pls add deployed live link

**Review by ShoaibTamboli — COMMENTED** [review]
Reviewed button html and button CSS. Looks good to me.

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,101 @@
+button{
```
Comment (shubhmehra) [inline components/button/button.css]: please avoid using direct element targeting! you can use class instead.

**Review by shubhmehra — APPROVED** [review]
looks good to me approved!

**Comment by ooreally** [comment]
> Pls add deployed live link

https://deploy-preview-6--decora.netlify.app/components/button/button.html

File: components/button/button.css (hunk 1 above)
Comment (ooreally) [reply components/button/button.css]: Noted.

**Pull request merged**